tests/rt.conf

```diff
@@ -2,20 +2,24 @@
 # CCPP PROD tests                                                                                                                                                                     #
 #######################################################################################################################################################################################
 
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | hera.intel     | fv3         |
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | cheyenne.intel | fv3         |
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | cheyenne.gnu   | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | hera.intel     | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | cheyenne.intel | fv3         |
+COMPILE | 32BIT=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                   | standard    | cheyenne.gnu   | fv3         |
 
 RUN     | fv3_ccpp_gfs_v15p2                                                                                                             | standard    |                | fv3         |
+RUN     | fv3_ccpp_gfs_v15p2_coldstart                                                                                                   | standard    |                |             |
+RUN     | fv3_ccpp_gfs_v15p2_restart                                                                                                     | standard    |                |             |
```

Comment:
This test (fv3_ccpp_gfs_v15p2_restart) must be executed only after the previous one (fv3_ccpp_gfs_v15p2_coldstart) successfully finishes, because it depends on it. Without specifying this dependency explicitly (in the 6th column) parallel execution of regression tests using ecFlow of Rocoto will fail. See how this kind of dependency is specified for fv3_ccpp_regional_control and fv3_ccpp_regional_restart test in develop branch.